### DISCUSSION TITLE
Variable, Material, Element Types

### DISCUSSION THREAD

**Discussion by czadeh** [idaholab/moose#26175] [Q&A General] [answered]
Hello,

I am writing a research paper about a simulation I created using MOOSE and I need help with some details.

I know the variables are first order Lagrange by default and that is what I am using. 

However, I am not sure what type the material properties are by default. I am using ADMaterialProperty Real. Are these first order monomial by default?

Also, I am using GeneratedMeshGenerator to create a 3D mesh. In the documentation, I see that the default elem_type is linear for the requested dimension but I am still unsure of what that means. What type of element is default for 3D?

Lastly, what is the preferred way of citing MOOSE for a publication?

Thank you,
Cameron

**Comment by GiudGiud** [comment, marked as answer]
Hello

By default material properties are computed at quadrature points. they are not a particular finite element family, they dont require a projection onto shape functions.

However, when we output them to exodus, we project them on a constant monomial basis.

Default is Hex8 element in 3D. Second order (quadratic) element would be Hex20. 

The preferred way of citing MOOSE is to follow the instructions there (and cite the 2.0 paper)
https://mooseframework.inl.gov/moose/citing.html

Guillaume

**Reply by czadeh** [reply]
Thank you!
